src/apis/Spotify.js

```diff
@@ -7,9 +7,11 @@ const API_URL = 'https://api.spotify.com/v1'
 
 module.exports = class SpotifyAPI extends APIWrapper {
   constructor () {
-    super()
-    this.name = 'spotify'
-    this.envVars = ['SPOTIFY_CLIENT_ID', 'SPOTIFY_CLIENT_SECRET']
+    super({
+      name: 'spotify',
+      envVars: ['SPOTIFY_CLIENT_ID', 'SPOTIFY_CLIENT_SECRET']
+    })
+
     this.token = null
```

Comment:
Is this supposed to be here?

Reply:
I'm forgot2

Reply:
Ohh, my fork is outdated, spotify connection wasn't there